ogc-example/src/main/resources/application.conf

```diff
@@ -51,7 +51,7 @@ wcs = {
             description = "Geotrellis WCS Service"
             keywords = []
             profile = ["http://azavea.com/wcs-profile"]
-            fees = []
+            fees = ""
```

Comment:
ah no, should it be reverted?

Reply:
That field is not currently being used and i cut a release to test with the fixes included here - this change is a fix to compensate for a purescript upgrade. I'm investigating right now to determine how the decoder ought to change

Reply:
String is a better representation after all. It looks like `[]` was being parsed to `None` whereas purescript treats missing fields as `None`s
![image](https://user-images.githubusercontent.com/1977405/57543403-4ed25400-7322-11e9-8936-bb5274f3d3c4.png)

Reply:
wow gotcha